### PR TITLE
HOMS-94 Add logo

### PR DESCRIPTION
**Pull request by profx5**

### CODE REVIEW

File: app/views/layouts/_navigation.html.haml [hunk 1]
```diff
@@ -8,6 +8,10 @@
         %span.icon-bar
     .navbar-collapse
       %ul.nav.navbar-nav
+        %li.logo
+          %a(href="/")
```
Comment (sugarfree1) [inline app/views/layouts/_navigation.html.haml]: Use root_path helper method.

**Comment by profx5** [comment]
UPD

**Comment by profx5** [comment]
с root_path не зашло

File: app/views/layouts/_navigation.html.haml [hunk 2]
```diff
@@ -8,6 +8,10 @@
         %span.icon-bar
     .navbar-collapse
       %ul.nav.navbar-nav
+        %li.logo
+          = link_to list_orders_url  do
+            Hydra OMS
```
Comment (sugarfree1) [inline app/views/layouts/_navigation.html.haml]: Add to translations

**Comment by profx5** [comment]
UPD

**Pull request merged**